test/wadler_test.rb

```diff
@@ -610,4 +610,40 @@ def altshow(out)
       check_roundtrip(100, expected, builder_block)
     end
   end
+
+  describe 'must not break if no group was given' do
+    it 'must work with only a string' do
+      out = Oppen::Wadler.new
+      out.text 'Hello World'
+      _(out.output).must_equal 'Hello World'
+    end
+
```

Comment:
```suggestion
  describe 'break if no group was given' do
    it 'works with a string only' do
      out = Oppen::Wadler.new
      out.text 'Hello World'
      _(out.output).must_equal 'Hello World'
    end

```

Let's use this phrasing from now on for tests. This is what minitest does anyway.
I will add a ticket to refactor the remainder of tests (outside this PR).